CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# CBL-Mariner Changelog
+
+All notable changes to this project will be documented in this file. For each notable change , please create an entry under the "Unreleased" header in a relevant category with a link to the related PR. If there is no relevant category present, please add one as appropriate.
+
+Suggested categories: "Security Fixes", "Package Additions", "Package Version Updates", "Package Test Fixes", "Toolkit Changes".
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+### [Unreleased]
+
+#### Security Fixes
+
+- CVE-2021-20208 in `cifs-utils` ([#910](https://github.com/microsoft/CBL-Mariner/pull/917))
+- CVE-2021-28965 in `ruby` ([#911](https://github.com/microsoft/CBL-Mariner/pull/911))
+- CVE-2021-27419 in `uclibc` ([#917](https://github.com/microsoft/CBL-Mariner/pull/917))
+
+#### Package Additions
+
+- `python3-Cython` ([#924](https://github.com/microsoft/CBL-Mariner/pull/924))
+
+#### Package Version Updates
```

Comment:
Considering that we want to be diligent about timely CVE fixes and mention these in our release notes, I'd suggest another section called "CVE fixes" or similar where for each updated package we'd have a list of fixed CVEs. Example:

```
#### CVE fixes

- `go`: CVE-AAAA-BBBBB (#CCC), CVE-XXXX-YYYYY (#ZZZ), ...
```

Reply:
I agree, but is this not covered by the "Security Fixes" sections?